packages/probing-api/src/utilities/enumUtils.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  EserviceInteropState,
+  eserviceInteropState,
+  eserviceMonitorState,
+  responseStatus,
+  EserviceStatus,
+  EserviceMonitorState,
+  EServiceContent,
+  EServiceProbingData,
+} from "pagopa-interop-probing-models";
+import { config } from "./config.js";
+
+const isBefore = (responseReceived: string, lastRequest: string) => {
+  return new Date(responseReceived).getTime() < new Date(lastRequest).getTime();
+};
+
+const minutesDifferenceFromCurrentDate = (lastRequest: string) => {
+  return (new Date().getTime() - new Date(lastRequest).getTime()) / (1000 * 60);
+};
+
+export function fromECToMonitorState(
+  data: EServiceContent
+): EserviceMonitorState {
+  return handleState({
+    state: data.state,
+    probingEnabled: data.probingEnabled,
+    lastRequest: data.lastRequest,
+    responseReceived: data.responseReceived,
+    pollingFrequency: data.pollingFrequency,
+    responseStatus: data.responseStatus
+  });
```

Comment:
isn't this equal to
```suggestion
  return handleState(data);
```
?